### PR TITLE
Wire up dataset controls

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: viewer/core/static/js/app/actions/url-actions.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  };
+}
+
+export function updateQueryParameter(key, newValue) {
+  return {
+    type: types.UPDATE_QUERY_PARAMETER,
+    key,
+    newValue,
+  };
+}
```
Comment (spasovski) [inline viewer/core/static/js/app/actions/url-actions.js]: I haven't seen the rest and *this* I love! Exactly what I was hoping we'd get to.

File: viewer/core/static/css/dataset-control.styl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.dataset-selection {
+  width: 150px;
```
Comment (spasovski) [inline viewer/core/static/css/dataset-control.styl]: Remove this please I've fixed it differently in something that will follow this patch.

File: viewer/core/static/js/app/components/containers/app-container.js [hunk 3]
```diff
@@ -93,8 +117,9 @@ class AppContainer extends React.Component {
 
 const mapStateToProps = function(store, ownProps) {
   return {
-    subgroups: store.subgroupsState.subgroups,
+    datasets: store.datasetState.datasets,
     metricMetadata: store.metricMetadataState.metadata,
+    subgroups: store.subgroupsState.subgroups,
```
Comment (spasovski) [inline viewer/core/static/js/app/components/containers/app-container.js]: subgroups are in datasets anyway no? It's fine to granularize just wanted to check.

**Comment by spasovski** [comment]
Looks great! This gets us mostly there just need to rewire the cohort toggles. r+wc

File: viewer/core/static/css/dataset-control.styl (hunk 2 above)
Comment (openjck) [inline viewer/core/static/css/dataset-control.styl]: Sure thing

File: viewer/core/static/js/app/components/containers/app-container.js (hunk 3 above)
Comment (openjck) [inline viewer/core/static/js/app/components/containers/app-container.js]: Ah, great catch. I'll rewrite this. Thanks.

Comment (openjck) [inline viewer/core/static/js/app/components/containers/app-container.js]: I was thinking of re-doing the subgroup API stuff to be more like your dataset approach but you're absolutely right that we don't need both.

Comment (openjck) [inline viewer/core/static/js/app/components/containers/app-container.js]: Actually, maybe I'll follow up with this. It's a little messy and I don't want to risk breaking the toggles.

**Pull request merged**